test/NuGet.Clients.Tests/NuGet.CommandLine.Test/RestoreNETCoreTest.cs

```diff
@@ -9488,7 +9484,7 @@ public async Task RestoreNetCore_CPVMProject_RemovedCentralTransitiveDependency_
         }
 
         [Fact]
-        public async Task RestoreNetCore_CPVMProject_MoveTransitiveDependnecyToCentralFile_FailsRestore()
+        public async Task RestoreNetCore_CPVMProject_MoveTransitiveDependencyToCentralFile_FailsRestore()
```

Comment:
In another comment, you said you didn't want to rename a test because you'd lose test history. Is renaming it here intentional?